tests/integration/test_random_permsort.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import numpy as np
+import pytest
+
+import cunumeric as num
+
+
+def test_permutation_int():
+    count = 1024
+    p = num.random.permutation(count)
+    p.sort()
+    assert num.linalg.norm(p - np.arange(count)) == 0.0
+
+
+def test_permutation_array():
+    count = 1024
+    x = num.arange(count)
+    p = num.random.permutation(x)
+    assert num.linalg.norm(x - p) != 0.0
+    p.sort()
+    assert num.linalg.norm(x - p) == 0.0
+
+
+def test_shuffle():
+    count = 16
+    p = num.arange(count)
+    x = num.arange(count)
+    num.random.shuffle(x)
+    assert num.linalg.norm(x - p) != 0.0
+    x.sort()
+    assert num.linalg.norm(x - p) == 0.0
+
+
+def test_choice_1(maxvalue=1024, count=42):
+    a = num.random.choice(maxvalue, count)
+    assert len(a) == count
+    assert num.amax(a) <= maxvalue
+    assert num.amin(a) >= 0
+
+
+def test_choice_2(maxvalue=1024, count=42):
```

Comment:
What is the benefit of making these parameters, without using `parametrize` or some other way to ever set them? It's not wrong, per se, but it is unusual, so someone running across the code in the future might be confused by it. Another option here might be to use a class for grouping and also defining the common params in one place:
```python
class Test_choice:

    maxvalue = 1024
    count = 16

    def test_1(self):
        a = num.random.choice(self.maxvalue, self.count)
        assert len(a) == count
        assert num.amax(a) <= maxvalue
        assert num.amin(a) >= 0

    def test_2(self):
        a = num.random.choice(self.maxvalue, self.count, False)
        ...

````